Source/RunActivity/RunActivity.csproj

```diff
@@ -205,7 +205,7 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="..\3rdPartyLibs\openal-soft\Win32\OpenAL32.dll">
-      <Link>Native\X86\OpenAL32.dll</Link>
+      <Link>Native\AnyCPU\OpenAL32.dll</Link>
```

Comment:
this seems wrong (AnyCPU). OpenAL32 has 32bit version and a 64bit version, though both are named OpenAL32. They are both copied to output so the corresponding version can be located and loaded at runtime, which is set in Source/RunActivity/Program.cs expecting the dlls in folder x86 for 32bit and x64 for 64bit:
https://github.com/openrails/openrails/blob/8d428b8cb38456442d04374d5a67461f5598074e/Source/RunActivity/Program.cs#L60-L61

with your change the x86 version is copied to a folder AnyCPU, and can't be found at runtime:
http://www.elvastower.com/forums/index.php?/topic/35959-problem-running-or-after-upgrading/

Reply:
Thanks, Steffen.

This PR is now marked as not-for-unstable and I'll test out your change tomorrow before unmarking it.

Reply:
All done. I think this bad change was the victim of a global search and replace.